vars/updatecli.groovy

```diff
@@ -8,9 +8,9 @@ def call(userConfig = [:]) {
     config: './updatecli/updatecli.d',      // Config manifest (file or directory) for updatecli
     values: './updatecli/values.yaml',      // Values file used by updatecli
     updatecliAgentLabel: 'jnlp-linux-arm64', // Label to select the Jenkins node (agent)
-    cronTriggerExpression: '',               // Enables cron trigger if specified
+    cronTriggerExpression: '',              // Enables cron trigger if specified
     credentialsId: 'github-app-updatecli-on-jenkins-infra', // GitHub credentials
-    version: ''                               // Custom updatecli version (e.g. '0.92.0' or '0.86.0-rc.1')
+    runInCurrentAgent: false                // New option: if true, run updatecli in the current node
```

Comment:
```suggestion
    version: '' // Custom updatecli version (e.g. '0.92.0' or '0.86.0-rc.1')
    runInCurrentAgent: false                // New option: if true, run updatecli in the current node
```

It is presnt on the main branch (https://github.com/jenkins-infra/pipeline-library/blob/ba0fca6efb8274f1dfe4b6d05d2e7dfb29481d37/vars/updatecli.groovy#L13).
Sounds like an issue with your rebase / conflicts ;)